/**:
  ros_parameters:
    led_gpio_pin: 37
    signal_gpio_pin: 40
